docs/examples/tensorflow/tensorflow-resnet50-various-readers.ipynb

```diff
@@ -255,12 +283,14 @@
       "RUN: MXNetReaderPipeline\n",
       "OK : MXNetReaderPipeline\n",
       "RUN: TFRecordPipeline\n",
-      "OK : TFRecordPipeline\n"
+      "OK : TFRecordPipeline\n",
+      "RUN: CaffeReadPipeline\n",
+      "OK : CaffeReadPipeline\n"
      ]
     }
    ],
    "source": [
-    "pipe_types = [FileReadPipeline, MXNetReaderPipeline, TFRecordPipeline]\n",
+    "pipe_types = [FileReadPipeline, MXNetReaderPipeline, TFRecordPipeline, CaffeReadPipeline]\n",
```

Comment:
Nit: should these be listed in (so as to be executed in) the same order as they appear in the source above?

Reply:
No required, but it may look better the way you suggested. Done